source/authentication.rst

```diff
@@ -28,9 +28,13 @@ LDAP authentication using SASL
    :local:
    :depth: 1
 
+
 Overview
 ========
 
+LDAP authentication using |SASL| means that both the client and the server establish a SASL session using the SASL library. Then authentication (bind) requests are sent to the LDAP server through the SASL authentication daemon (``saslauthd``) that acts as a remote proxy for the ``mongod`` server.
```

Comment:
I'd rephrase this - it's not LDAP auth using SASL, it's rather auth VIA saslauthd daemon or LDAP proxy authentication (as mongodb official doc suggests)